### ISSUE TITLE
Official release date for notify

### ISSUE THREAD

**Issue by jong-eatsa** [twilio/twilio-python#380]
It looks like Python support for the notify service is still in alpha (6.4.3.a1).

1) When will it officially be available? It's been in alpha for a while.
2) How safe is it to use the alpha version in production? I've tested and verified its functionality.
3) If there are any issues / bugs preventing its general availability, what are they?

P.S. Would you consider the ruby notify service (also available only as a release candidate) to be more mature? Found [this](https://github.com/twilio/twilio-ruby/issues/218). Our Ruby backend currently uses rc24.

**Comment by dougblack** [comment]
Hey @jong-eatsa! First off, in the latest [6.5.0](https://github.com/twilio/twilio-python/releases/tag/6.5.0) release, we included all of the Beta and Developer Preview products for the first time. As part of this, we've discontinued the alpha release line so going forward you'll want to switch to the normal release line.

As far as your numbered questions, I'll have to refer you to [this article](https://support.twilio.com/hc/en-us/articles/115002413087-Twilio-Beta-product-support) which details what being a Beta product means.

All six Twilio helper libraries consume the same underlying APIs, so the maturity (GA vs Beta vs Developer Preview) of the underlying products will always be the same across different helper libraries. The Ruby release candidates line existed as we iterated towards version 5 of the library. We've finished that iteration, and we encourage you to give `twilio-ruby` version `5.1.0` a try on your backend, it contains several bug fixes and product updates you might find useful.
